src/test/java/uk/ac/ebi/subs/agent/converters/SampleConverterTest.java

```diff
@@ -61,7 +63,9 @@ public void convertFromBsdSample() {
         uk.ac.ebi.biosamples.model.
                 Sample conversionBack = toBsdSample.convert(conversion);
 
-        Assert.assertEquals(bsdSample, conversionBack);
+        assertEquals(bsdSample.getAttributes(), conversionBack.getAttributes());
```

Comment:
I think, this is a subset of the next assert. Do we need it?

Reply:
🤔  you are probably right, but it doesn't hurt in any case.